sdv/constraints/tabular.py

```diff
@@ -379,3 +381,73 @@ def reverse_transform(self, table_data):
         table_data[self._column] = self._formula(table_data)
 
         return table_data
+
+
+class Rounding(Constraint):
+    """Round a column based on the specified number of digits.
+
+    Args:
+        column (list[str]):
+            Name of the column(s) to round.
+        digits (int or dict[str->int]):
+            How much to round each column. If an `int` is provided, all columns
+            will be rounded to that number of digits. If a `dict` that maps column
+            to digits is provided, each column will be represented to the specified
+            number of digits.
+        handling_strategy (str):
+            How this Constraint should be handled, which can be ``transform``
+            or ``reject_sampling``. Defaults to ``transform``.
+        tolerance (int):
+            How many differences in decimal places we will tolerante when
+            reject sampling.
+    """
+
+    def __init__(self, columns, digits, handling_strategy='transform', tolerance=1):
+        self._columns = columns
+        self._digits = digits
+        self._tolerance = tolerance
+        super().__init__(handling_strategy=handling_strategy, fit_columns_model=False)
+
+    def is_valid(self, table_data):
+        """Determine if the data satisfies the rounding constranit.
+
+        Args:
+            table_data (pandas.DataFrame):
+                Table data.
+
+        Returns:
+            pandas.Series:
+                Whether each row is valid.
+        """
+        table_data = table_data.copy()
+        d_places = table_data[self._columns].applymap(
+            lambda d: -1 * decimal.Decimal(str(d)).as_tuple().exponent)
```

Comment:
what happens if the rounding is a negative number? If a user creates the constraint with the `digits` value set to -2, I don't think this strategy works because the `Decimal` exponent property has a minimum of 0 I think

Reply:
Ah good point! I updated this map method to find the number of trailing zeros if there are no decimals.